### DISCUSSION TITLE
publicFolder not foud

### DISCUSSION THREAD

**Discussion by SalvatoreDag** [tinacms/tinacms#3974] [Help]
Hi, I'm trying to configure Tina with my Jekyll project but it keeps giving me the error  

> "Cannot read properties of undefined (reading 'publicFolder')" 

when I run the command  

> "npx tinacms dev -c "jekyll serve""

. This is my config.js

`import { defineConfig } from 'tinacms';

export default defineConfig({
    publicFolder: './', // Directory pubblica personalizzata del tuo progetto Jekyll
    schema: {
        collections: [
          {
            label: 'Blog Posts',
            name: 'post',
            path: 'content/posts',
            fields: [
              {
                type: 'string',
                label: 'Title',
                name: 'title',
              },
              {
                type: 'string',
                label: 'Post Body',
                name: 'body',
                isBody: true,
              },
            ],
          },
        ],
      },
});`
Am I doing something wrong?

**Comment by jeffsee55** [comment]
`publicFolder` should be inside the `build` key. I believe for Jekyll sites the `_site` folder is where assets should be built to
```
export default defineConfig({
  build: {
    publicFolder: '_site',
    outputFolder: 'admin'
  }
```
